### PR TITLE
Circle two

### PR DESCRIPTION
**Pull request by MiyamotoAkira**
Moving to circleci 2.0

### CODE REVIEW

**Comment by MiyamotoAkira** [comment]
Cannot be merged until the build happens properly

File: Dockerfile [hunk 1]
```diff
@@ -3,13 +3,19 @@ FROM docker.io/ruby:2.3.1
...
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
 RUN apt-get install -y nodejs
 RUN npm install pug -g
 RUN apt-get clean
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen
 RUN dpkg-reconfigure --frontend=noninteractive locales
 
+# This steps are needed to have aws cli installed on the system
+RUN apt-get -y install python3 python-pip python-dev
+RUn pip install --upgrade pip
```
Comment (richardjwild) [inline Dockerfile]: `RUN`

Comment (MiyamotoAkira) [reply Dockerfile]: done

**Review by richardjwild — APPROVED** [review]
Looks gouda to me.

**Pull request merged**